js/src/enum.ts

```diff
@@ -201,6 +201,7 @@ export enum Type {
     SparseUnion = -24,
     IntervalDayTime = -25,
     IntervalYearMonth = -26,
+    LargeUtf8 = -27,
```

Comment:
Is this right?

Reply:
>  * **Note**: Only enum values 0-17 (NONE through Map) are written to an Arrow IPC payload.

Seems like `LargeUtf8` would be given a positive number then

Reply:
I think it should be 20, according to the flatbuffers file that you get when you build the package (also the comment about 0-17) is probably outdated.

Reply:
Fixed it but we should probably do https://github.com/apache/arrow/issues/39048.